x-pack/test/api_integration/apis/maps/get_grid_tile.js

```diff
@@ -27,6 +27,11 @@ export default function ({ getService }) {
         .responseType('blob')
         .expect(200);
 
+      expect(resp.headers['content-encoding']).to.be('gzip');
+      expect(resp.headers['content-disposition']).to.be('inline');
+      expect(resp.headers['content-type']).to.be('application/x-protobuf');
+      expect(resp.headers['cache-control']).to.be('public, max-age=3600');
+
       const jsonTile = new VectorTile(new Protobuf(resp.body));
```

Comment:
the `.body`-getter already transparently unzips the response. Removing the gzip content-encoding header will (correctly) break the functional tests because the contents are indeed gzipped.